### DISCUSSION TITLE
What starts with `p`, ends with `e` and contains thousands of letters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#697] [Galaxy Trivia] [answered]
*answer with emoji*

**Comment by peterbaccaro** [comment, marked as answer]
🏤 **Post Office**

The words “post office” start with the letter “p”, and end with the letter “e”, but the actual post office contains thousands of “letters” (letters that people write and mail to one another).

**Comment by nsafarova** [comment]
The answer to your riddle is "Post Office" 📮.
